From 2cd194ee980fb4481999cda682ad03cd3479db9b Mon Sep 17 00:00:00 2001
From: Xing Zheng <zhengxing@rock-chips.com>
Date: Thu, 17 May 2018 09:13:36 +0800
Subject: [PATCH] arm64: dts: rockchip: add some properties for RK3308 EVB

This patch add rockchip,card-name, and add rockchip,codec-hp-det
if with acodec for vad-sound.

Change-Id: Ib81651f9b52933f24257a42e977a5382da745df8
Signed-off-by: Xing Zheng <zhengxing@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts     | 2 ++
 arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v11.dts     | 2 ++
 arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v10.dts | 1 +
 arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v11.dts | 1 +
 arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v10.dts | 1 +
 arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v11.dts | 1 +
 6 files changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
index 9ed61eecb858..33f001cee701 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v10.dts
@@ -15,6 +15,8 @@
 	vad_acodec_sound: vad-acodec-sound {
 		status = "disabled";
 		compatible = "rockchip,multicodecs-card";
+		rockchip,card-name = "rockchip,rk3308-vad";
+		rockchip,codec-hp-det;
 		rockchip,mclk-fs = <256>;
 		rockchip,cpu = <&i2s_8ch_2>;
 		rockchip,codec = <&acodec>, <&vad>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v11.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v11.dts
index c52f04f7500b..f16147de31a7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v11.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-amic-v11.dts
@@ -14,6 +14,8 @@
 	vad_acodec_sound: vad-acodec-sound {
 		status = "disabled";
 		compatible = "rockchip,multicodecs-card";
+		rockchip,card-name = "rockchip,rk3308-vad";
+		rockchip,codec-hp-det;
 		rockchip,mclk-fs = <256>;
 		rockchip,cpu = <&i2s_8ch_2>;
 		rockchip,codec = <&acodec>, <&vad>;
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v10.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v10.dts
index 54a978aabbe9..80370f059602 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v10.dts
@@ -29,6 +29,7 @@
 	vad-sound {
 		status = "disabled";
 		compatible = "rockchip,multicodecs-card";
+		rockchip,card-name = "rockchip,rk3308-vad";
 		rockchip,cpu = <&i2s_8ch_0>;
 		rockchip,codec = <&dummy_codec>, <&vad>;
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v11.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v11.dts
index bf27d526325b..7f633a5e8f25 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v11.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-i2s-v11.dts
@@ -28,6 +28,7 @@
 	vad-sound {
 		status = "disabled";
 		compatible = "rockchip,multicodecs-card";
+		rockchip,card-name = "rockchip,rk3308-vad";
 		rockchip,cpu = <&i2s_8ch_0>;
 		rockchip,codec = <&dummy_codec>, <&vad>;
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v10.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v10.dts
index f8501a51fd38..23013b9cd6fc 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v10.dts
@@ -27,6 +27,7 @@
 	vad-sound {
 		status = "disabled";
 		compatible = "rockchip,multicodecs-card";
+		rockchip,card-name = "rockchip,rk3308-vad";
 		rockchip,cpu = <&pdm_8ch>;
 		rockchip,codec = <&dummy_codec>, <&vad>;
 	};
diff --git a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v11.dts b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v11.dts
index e0d9557eb420..0f2416551b41 100644
--- a/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v11.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3308-evb-dmic-pdm-v11.dts
@@ -26,6 +26,7 @@
 	vad-sound {
 		status = "disabled";
 		compatible = "rockchip,multicodecs-card";
+		rockchip,card-name = "rockchip,rk3308-vad";
 		rockchip,cpu = <&pdm_8ch>;
 		rockchip,codec = <&dummy_codec>, <&vad>;
 	};
-- 
2.35.3

